python-examples/README.md

```diff
@@ -4,8 +4,254 @@ Termhub in 5 Minutes: python Tutorial
+
+```
+pytest tests/test_get_concept_by_code_with_explicit_include_param.py::TestGetConceptWithIncludeParam::test_get_concept_by_code_with_include_param
+```
+
+[Back to Top](#top)
+
+<a name="get-concept-relationships"/>
+
+### Get concept relationships by code
+
+Get concept relationships for a terminology and code. In this case it resolves
+relationships that originate "from" this concept code and contains information about
+the concepts those relationships point "to" on the other side. For example, a child
+concept pointing to its parent.
+
+```
+pytest tests/test_get_concept_relationships_by_code.py::TestGetConceptRelationshipByCode::test_get_concept_relationships_by_code
+```
+
+[Back to Top](#top)
+
+<a name="get-concept-inverse-relationships"/>
+
+### Get concept inverse relationships by code
+
+Get concept inverse relationships for a terminology and code. In this case it resolves
+relationships that point "to" this concept code and contains information about concepts
+those relationships originate "from" on the other side. For example, a parent concept
+being pointed to from a child.
+
+```
+pytest tests/test_get_concept_inverse_relationships_by_code.py::TestGetConceptInverseRelationshipsByCode::test_get_concept_inverse_relationships_by_code
+```
+
+[Back to Top](#top)
+
+<a name="get-treepos"/>
+
+### Get concept tree positions
+
+Get concept tree positions for a terminology and code. For classification
+hierarchies, you would expect to see just a single tree position. But for
+more complex poly-hierarchies you'd likely expect to see multiple tree positions -
+each one with a different path to the root concept.
+
+```
+pytest tests/test_get_concept_trees.py::TestGetConceptTrees::test_get_concept_trees
+```
+
+[Back to Top](#top)
+
+<a name="find-concepts"/>
+
+### Find concepts by search term
+
+Find concepts matching a search term within a specified terminology. This
+example uses paging to get only the first 5 results.
+
+```
+pytest tests/test_find_concept_by_search_term.py::TestFindConceptBySearchTerm::test_get_concept_by_search_term
+```
+
+[Back to Top](#top)
+
+<a name="find-concepts"/>
+
+### Find concepts by search term with explicit include parameter
+
+This is the same as the example above but with the use of an explicit include
+parameter to show that additional data can be loaded with searches.
+
+```
+pytest tests/test_find_concepts_by_search_term_with_include_param.py::TestFindConceptsByTermWithIncludeParam::test_get_concept_by_query_and_include_param
+```
+
+[Back to Top](#top)
+
+<a name="find-concepts-expr"/>
+
+### Find concepts by search term and expression
+
+Find concepts matching a search term within a specified terminology and constrain
+the search results by an expression. This example uses paging to get only the first
+10 results.
+
+NOTE: the expression we are using is <<64572001 (descendants-or-self of the "Disease"
```

Comment:
This seems very helpful to me.  It tells me what to expect as the result since it isn't visible from the command.  But if you don't do it on all of the commands, don't do it here either.  Additionally, I think you should explain to the user that to get customizable results, they'll have to update the #SETUP code block in each script.

Reply:
This is copied from the curl README. If it's there, we want to keep the parity between the files